org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -17,28 +17,23 @@ define(function () {
      */
     function open () {
       var modalInstance;
-      var lockScrollStyle = 'overflow: hidden; height: 100%; width: 100%;';
+      var lockScrollStyle = ';overflow: hidden; height: 100%; width: 100%';
```

Comment:
Why `;` is placed in front  as in  ";overflow: hidden; height: 100%; width: 100%' ?

Reply:
Good question! This func populates style stack to an existing style stack in the `style` attribute of an HTML node (in our case, `body` and `html`). Style attribute can have the following value cases:

1) *no style attribute*
2) style=""
3) style="width: 50%"
4) style="width: 50%;"

Lets say we populate "width: 100%;" style. After style population we will have:

1) style="width:100%;" - good
2) style="width:100%;" - good
3) style="width: 50%width: 100%" - fail
4) style="width: 50%;width: 100%" - good

Now if we populate ";width: 100%" style. After style population we will have:

1) style=";width:100%" - not perfect, but works
2) style=";width:100%" - not perfect, but works
3) style="width: 50%;width: 100%" - good
4) style="width: 50%;;width: 100%" - not perfect, but works

Reply:
@igor Thanks! i was worried about the case:

> 4) style="width: 50%;;width: 100%"